src/reducers/AddSiteReducer.js

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash';
+
+import {
+    LATITUDE_TEXT_UPDATED,
+    LONGITUDE_TEXT_UPDATED,
+    SITE_TITLE_TEXT_CHANGED,
+    SITE_DESCRIPTION_TEXT_CHANGED,
+    SITE_DIRECTIONS_TEXT_CHANGED,
+    SITE_NEAREST_TOWN_TEXT_CHANGED,
+    SITE_ACCESSIBILITY_OPTION_CHANGED,
+    SITE_FACILITIES_OPTION_CHANGED,
+    ADD_SITE_FIELDS_RESET
+} from '../actions/types';
+
+import {campsite} from '../locale.en';
+
+const {campsite_form: {accessibility_options, facilities_options}} = campsite;
+
+const INITIAL_STATE = {
+    latitudeText: '',
+    longitudeText: '',
+    siteTitleText: '',
+    siteDescriptionText: '',
+    siteDirectionsText: '',
+    siteNearestTownText: '',
+    accessibilityOption: accessibility_options.paved_road,
+    facilitiesOption: facilities_options.full_service
+};
+
+const removeNonNumbers = (text) => {
+    return text.replace(/[^0-9.-]/g, '');
+};
+
+export default (state = INITIAL_STATE, action) => {
+    const {type, payload} = action;
+
+    switch (type) {
+
+        case LATITUDE_TEXT_UPDATED:
+            const {latitudeText} = payload;
+            const cleanLatText = removeNonNumbers(latitudeText);
+
+            return {...state, latitudeText: cleanLatText};
+
+        case LONGITUDE_TEXT_UPDATED:
+            const {longitudeText} = payload;
+            const cleanLongText = removeNonNumbers(longitudeText);
+
+            return {...state, longitudeText: cleanLongText};
```

Comment:
`LATITUDE_TEXT_UPDATED` and `LONGITUDE_TEXT_UPDATED` have validations to make sure it's a valid number, but the rest of the fields just pass along whatever they received.